### PR TITLE
Create updated Varnish 6 image, and rename existing to 5

### LINKED ISSUE #43: Support more recent non-EOL Varnish images (6.0/6.5)
Currently, we use the varnish installed from alpine:3.7 (https://github.com/uselagoon/lagoon-images/blob/main/images/varnish/Dockerfile#L18), which is `5.2.1` :scream: 

I have looked at the versions of varnish available on alpine packages, and cross-referenced them against supported varnish versions (https://varnish-cache.org/releases/) - to discover that none of the alpine versions of varnish are actually supported.

In order to provide a supported version of varnish, we should switch to the upstream varnish docker hub images (built on buster-slim - https://hub.docker.com/_/varnish)

In order to support varnish 6.0/6.5, we will also need to change the versions of the included vmods
* libvmod-dynamic has 6.0 and 6.5 releases
* libvmod-bodyaccess has been subsumed by varnish/varnish-modules, which has 6.0 and 6.5 releases

In addition, some of the vcl_hit syntax in our drupal.vcl - return(miss) - has been deprecated in 6.2 (https://varnish-cache.org/docs/6.2/whats-new/upgrading-6.2.html#whatsnew-upgrading-vcl-2019-03).

We should look to create 6.5-based images for Varnish (and selected variants), and maintain the existing versions for 5.2

### PR DESCRIPTION
**Pull request by tobybellwood**
This PR adds in a new image for Varnish 6.5, and renames the current varnish image to specify that it's varnish 5.2 (and is EOL).

The new 6.5-based image is built from the upstream varnish images, built on buster-slim (instead of alpine).

I don't want to force update anybody from 5 to 6 (especially if they extend our configs), hence the naming change (similar to how we did postgres and redis images)

To update the version of bodyaccess, I've migrated to https://github.com/varnish/varnish-modules - which may also allow the inclusion of other modules if wanted.

Closes #43

### CODE REVIEW

File: images/varnish-drupal/drupal.vcl [hunk 1]
```diff
@@ -279,7 +279,7 @@ sub vcl_hit {
       return (deliver);
     } else {
       # No candidate for grace. Fetch a fresh object.
-      return (miss);
+      return (pass);
```
Comment (tobybellwood) [inline images/varnish-drupal/drupal.vcl:282]: @seanhamlin - apparently `miss` never worked - is pass appropriate here?

OR should we implement the recommended steps from here - https://varnish-cache.org/docs/6.2/whats-new/upgrading-6.2.html#whatsnew-upgrading-vcl-2019-03

Comment (seanhamlin) [reply images/varnish-drupal/drupal.vcl:282]: Yeah, the docs seem clear on what is needed to reproduce this behaviour in 6.2, with the `return (restart)`.

Comment (tobybellwood) [reply images/varnish-drupal/drupal.vcl:282]: I'll have another look at the `vcl_recv{}` section to incorporate the code - I'll reach out for vcl ninja skills...

**Comment by tobybellwood** [comment]
In line with other non-base images, I've renamed these back to Major Version only - i.e. varnish-5 and varnish-6

**Pull request merged**